README.md

```diff
@@ -22,7 +22,9 @@
 
     $ cordova plugin add cordova-plugin-idfa
 
-Use variable `ANDROID_PLAY_ADID_VERSION` to override dependency version on Android.
+Use variable `ANDROID_PLAY_ADID_VERSION` to override dependency version on Android:
+
+    $ cordova plugin add cordova-plugin-idfa --variable ANDROID_PLAY_ADID_VERSION='16.+'
```

Comment:
Good comment. I should probably add such example into each repo because people are asking a lot about this.